<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormQuery</class>
 <widget class="QWidget" name="FormQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Query HMDB</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
     <item>
      <layout class="QVBoxLayout" name="frameLeft" stretch="1,10,0">
       <item>
        <layout class="QVBoxLayout" name="frameSearchBox">
         <item>
          <widget class="QLabel" name="labelSearchBy">
           <property name="text">
            <string>Search by:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>ID</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Name</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Molecular Mass</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>MS/MS Spectrum</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="frameSearchOption">
         <item>
          <widget class="QStackedWidget" name="widgetSearchOption">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pageOptionID">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="labelQueryID">
               <property name="text">
                <string>HMDB ID:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="textQueryID"/>
             </item>
             <item>
              <widget class="QPushButton" name="buttonQueryID">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageOptionName">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLineEdit" name="textQueryName"/>
             </item>
             <item>
              <widget class="QPushButton" name="buttonQueryName">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageOptionMass">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="frameMassInput">
               <item>
                <widget class="QRadioButton" name="radioAvgMass">
                 <property name="text">
                  <string>A&amp;verage mass</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioMonoMass">
                 <property name="text">
                  <string>&amp;Monoisotopic mass</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="textMass"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="buttonQueryMass">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="frameMassToleranceInput">
               <item>
                <layout class="QHBoxLayout" name="frameLabelMassTolerance">
                 <item>
                  <widget class="QLabel" name="labelMassTolerance">
                   <property name="text">
                    <string>Tolerance:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelMassToleranceUnit">
                   <property name="text">
                    <string>(ppm)</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLineEdit" name="textMassTolerance">
                 <property name="text">
                  <string>100</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="checkRelativeTolerance">
                 <property name="text">
                  <string>Use relative value</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageOptionMSMS">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QFrame" name="frameSelectMSMS">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="frameSelectMSMSFile">
                  <item>
                   <widget class="QRadioButton" name="radioMSMSFromFile">
                    <property name="text">
                     <string>Fro&amp;m file:</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="buttonSelectMSMSFile">
                    <property name="text">
                     <string notr="true">...</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QLineEdit" name="textMSMSFile"/>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioMSMSFromInput">
                  <property name="text">
                   <string>From inp&amp;ut:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="textMSMSPeaks">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonOptionQueryMSMS">
               <property name="text">
                <string>Options</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonQueryMSMS">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="frameDatabase">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="labelSetDatabase">
             <property name="text">
              <string>Use database:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonSetDatabase">
             <property name="text">
              <string notr="true">...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="textDatabase">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="frameRight">
       <item>
        <widget class="FrameQueryResult" name="frameResult">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="frameStatus">
         <item>
          <widget class="QLabel" name="labelStatus">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressQuery">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FrameQueryResult</class>
   <extends>QFrame</extends>
   <header>widgets/framequeryresult.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
